.github/bot-pr-base.sh

```diff
@@ -66,7 +66,7 @@ bot_get_all_changed_files() {
   while true; do
     # this api allows 100 items per page
     # github action uses `bash -e`. The last empty page will leads jq error, use `|| :` to ignore the error.
-    local pr_page_files=$(api_get "$pr_url/files?&per_page=100&page=${page}" | jq -er '.[] | .filename' || :)
+    local pr_page_files=$(api_get "$pr_url/files?&per_page=100&page=${page}" | jq -er '.[] | select(.status != "removed") | .filename' || :)
```

Comment:
Do you know whether the deleted file is always in the end of list?
if one pages contains 100 deleted files, it will stop earlier than we thought.

Reply:
Yes, but this would only happen if all files on a page are deleted, which is very unlikely. I was a bit too lazy to make it fully correct 😁

Reply:
Yes. it is pretty corner cases. I am totally fine with the current changes